### PR TITLE
Picker drawer screen accessibility updates

### PR DESCRIPTION
**Pull request by kameroli**
# **ChangeLog**

- Added a global variable to store the focused element by accessibility
- Add default behavior to drawer screen so it focus back on element that was tapped when the drawer is dismissed

### CODE REVIEW

File: Sources/Cocoa/Components/Picker/DrawerScreen/DrawerScreen.swift [hunk 1]
```diff
@@ -113,6 +114,10 @@ final public class DrawerScreen: NSObject {
         }
     }
 
+    func setCaller(caller: Any?) {
+        drawerCaller = caller
+    }
+
```
Comment (mstana) [inline Sources/Cocoa/Components/Picker/DrawerScreen/DrawerScreen.swift:120]: Are we able to get rid of this function? If I am not mistaken - the only access to `drawerCaller` is here in the class `DrawerScreen `

File: Sources/Cocoa/Components/Picker/DatePicker.swift [hunk 2]
```diff
@@ -47,17 +48,19 @@ final class DatePicker: DrawerScreen.Content, Appliable {
...
             $0.pickerView.date = date ?? Date()
+            $0.caller = caller
             $0.didChangeValue = callback
             if let configuration = configuration {
                 $0.pickerView.apply(configuration)
             }
         }
 
-        DrawerScreen.present(picker)
+        DrawerScreen.present(picker, caller: caller)
```
Comment (mstana) [inline Sources/Cocoa/Components/Picker/DatePicker.swift:64]: Just thought here:

We are setting `caller` to `DatePicker` and then we are populating the same caller to `DrawerScreen` - as a developer who sees this for the first time I have red flag here - Do we have a way how to merge these two? So we will not populate the same item twice? (maybe not, I haven't been digging into this)

File: Sources/Cocoa/Components/Picker/DrawerScreen/DrawerScreen.swift (hunk 1 above)
Comment (kameroli) [reply Sources/Cocoa/Components/Picker/DrawerScreen/DrawerScreen.swift]: Done!

File: Sources/Cocoa/Components/Picker/DatePicker.swift (hunk 2 above)
Comment (kameroli) [reply Sources/Cocoa/Components/Picker/DatePicker.swift]: Good catch, it was unnecessary to assign it twice.

**Pull request merged**